//////////////////////////////////////////////////////////////////////////
// ³é¿¨¹¦ÄÜÄ£¿é

HandleOpcode( CMSG_ALL_LOTTERY_FREE_REQ)

HandleOpcode( CMSG_USE_FREE_LOTTERY)

HandleOpcode( CMSG_LOTTERY_ONE)

HandleOpcode( CMSG_LOTTERY_TEN)

HandleOpcode( CMSG_LOOT_EQUIP_SHOP)

HandleOpcode( CMSG_EQUIP_PURIFY)
HandleOpcode( CMSG_EQUIP_PURIFY_RETRIEVAL)

HandleOpcode( CMSG_RESET_BUY_ITEM_CD)
HandleOpcode( CMSG_EQUIP_PURIFY_ITEM )